stl/inc/chrono

```diff
@@ -2229,7 +2270,140 @@ namespace chrono {
             return _Name;
         }
 
+        template <class _Duration>
+        _NODISCARD sys_info get_info(const sys_time<_Duration>& _Sys) const {
+            return _Get_info(_Sys.time_since_epoch());
+        }
+
+        template <class _Duration>
+        _NODISCARD local_info get_info(const local_time<_Duration>& _Local) const {
+            local_info _Info{};
+            _Info.first = _Get_info(_Local.time_since_epoch());
+
+            const sys_seconds _Local_sys{duration_cast<sys_seconds::duration>(_Local.time_since_epoch())};
+            const auto _Curr_sys = _Local_sys - _Info.first.offset;
+            if (_Info.first.begin != _Min_seconds && _Curr_sys < _Info.first.begin + days{1}) {
+                // get previous transition information
+                _Info.second = get_info(_Info.first.begin - seconds{1});

```

Comment:
Calling this twice does incur an extra cost of transforming a string and reloading a ICU calendar. An alternative solution is to have a new `__std__tzdb_get_local_info` function and do all this logic there. 

I originally had that solution but it added a lot more code complexity, particularly around allocations and error handling. In my mind the slight performance gain doesn't justify this corner case (happens 4/256 times because there is generally 2 transitions a year).

Reply:
If this is the desired change lets scope it to a future PR in `feature/chrono`